reconcile/aus/base.py

```diff
@@ -739,6 +784,49 @@ def verify_lock_should_skip(
     return False
 
 
+def _create_upgrade_policy(
+    next_schedule: str, spec: ClusterUpgradeSpec, version: str
+) -> AbstractUpgradePolicy:
+    if spec.cluster.is_rosa_hypershift():
+        return ControlPlaneUpgradePolicy(
+            cluster=spec.cluster,
+            version=version,
+            schedule_type="manual",
+            next_run=next_schedule,
+        )
+    return ClusterUpgradePolicy(
+        cluster=spec.cluster,
+        version=version,
+        schedule_type="manual",
+        next_run=next_schedule,
+    )
+
+
+def _calulate_node_pool_diffs(
+    ocm_api: OCMBaseClient, spec: ClusterUpgradeSpec, now: datetime
+) -> Optional[UpgradePolicyHandler]:
+    node_pools = get_node_pools(ocm_api, spec.cluster.id)
+    if node_pools:
+        for pool in sorted(node_pools, key=lambda x: x["id"]):
+            pool_version_id = pool.get("version", {}).get("id")
+            pool_version = get_version(ocm_api, pool_version_id)["raw_id"]
+            if semver.match(pool_version, f"<{spec.current_version}"):
+                next_schedule = (now + timedelta(minutes=6)).strftime(
```

Comment:
6 mins seems a magic number, any special requirement for it?

Reply:
Yes, ocm api requires at least 5 minutes in the future, adding an extra minute just for safety.

Reply:
oh, then maybe extract a const like `MIN_DELTA_MINUTES = 6` to make it explicit.

Reply:
Good idea, done!